src/coreclr/jit/emitriscv64.cpp

```diff
@@ -4155,12 +4156,11 @@ regNumber emitter::emitInsTernary(instruction ins, emitAttr attr, GenTree* dst,
         }
         else
         {
-#ifdef DEBUG
             printf("RISCV64-Invalid ins for overflow check: %s\n", codeGen->genInsName(ins));
-#endif
             assert(!"Invalid ins for overflow check");
```

Comment:
I think this change should be reverted. The JIT should never print to stdout in release builds, for any reason.